spec/Section 3 -- Type System.md

```diff
@@ -1778,3 +1790,36 @@ type ExampleType {
   oldField: String @deprecated(reason: "Use `newField`.")
 }
 ```
+
+
+### @specifiedBy
+
+```graphql
+directive @specifiedBy(
+  url: String!
+) on SCALAR
+```
+
+The `@specifiedBy` directive is used within the type system definition language
+to provide a URL for specifying the behaviour of custom
```

Comment:
I believe this project prefers American spelling.
Reasoning: `color` is in this project 74 times, compared to `colour` which is never used.

```suggestion
to provide a URL for specifying the behavior of custom
```

Reply:
Whoops, my mistake, I knew that and just forgot.